### ISSUE TITLE
[Feature Request]: User interface, silent errors if not connected

### ISSUE THREAD

**Issue by tinmarino** [puremourning/vimspector#83]
**Is your feature request related to a problem? Please describe.**
Press `f6` (Pause) or `f10` (next) without connection gets an error message (here for f10)
```
Error detected while processing function vimspector#StepOver:
line    1:
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home2/tourneboeuf/.vim/pack/bundle/opt/spector/python3/vimspector/debug_session.py", line 306, in StepOver
    if self._stackTraceView.GetCurrentThreadId() is None:
AttributeError: 'NoneType' object has no attribute 'GetCurrentThreadId'
```

**Describe the solution you'd like**
1. Hide the error message with:
```python
if self._stackTraceView is None or self._stackTraceView.GetCurrentThreadId() is None:
```
instead of
```python
self._stackTraceView.GetCurrentThreadId() is None:
```
2. Echo a nice little message `MessageUser` with a warning color asking him to connect first
3. Autoconnect ?

**Describe alternatives you've considered**
1. Do the first stepOver() is auto connecting if not connected ? 
2. Is is preferable to silence the error, and even no message

**Additional context**
My solution would be 1, 2, 3. I'd be glad to make those changes with your desired solution.

**Comment by puremourning** [comment]
Yeah this is a bug. I’ve know about it for ages but been too lazy to fix it. A simple fix is to just make the methods tolerate being called before connection. Im happy with silence or a little user message.

**Comment by puremourning** [comment]
This was fixed in #85 and #77
